src/main/java/edu/indiana/dlib/amppd/service/impl/JobServiceImpl.java

```diff
@@ -319,7 +320,7 @@ protected WorkflowInputs buildWorkflowInputs(WorkflowDetails workflowDetails, St
 		List<String> stepsChanged = new ArrayList<String>();		
 		
 		workflowDetails.getSteps().forEach((stepId, stepDef) -> {
-			if (StringUtils.startsWith(stepDef.getToolId(), HMGM_TOOL_ID_PREFIX) || StringUtils.equals(stepDef.getToolId(), APPLAUSE_TOOL_ID)) {
+			if (StringUtils.startsWith(stepDef.getToolId(), HMGM_TOOL_ID_PREFIX) || StringUtils.equals(stepDef.getToolId(), APPLAUSE_TOOL_ID) || StringUtils.equals(stepDef.getToolId(), COLLECTION_CONTACT_SHEET_TOOL_ID)) {
```

Comment:
Can we put APPLAUSE_TOOL_ID  and COLLECTION_CONTACT_SHEET_TOOL_ID into a static String list MGM_TOOL_IDS, and check with Collection.contains. we could have more in the future.

Reply:
Updated!